airbyte-integrations/connectors/source-zendesk-support/acceptance-test-config.yml

```diff
@@ -39,7 +39,7 @@ acceptance_tests:
       - config_path: "secrets/config.json"
         configured_catalog_path: "integration_tests/incremental_catalog.json"
         future_state:
-          future_state_path: "integration_tests/abnormal_state.json"
+          bypass_reason: "This test does not make sense using Concurrent CDK"
```

Comment:
This however no longer holds true in the concurrent framework because the final state we emit is either latest synced record of a date window, or the lower bound of the window. We don’t want to use the upper bound of the window because its possible we’ve missed delayed records in that window. This was fixed in the [PR](https://github.com/airbytehq/airbyte/pull/45180/files#diff-e1414d3b2f03677efb789279bea1737ab2437e04bd3c3afe3d817656220ccf86R56-R57). And this will now lead to the mentioned test always failing because instead of emitting the date impossibly far in the future, we emit the bottom of the window which is typically the start date.

We intend to delete this test eventually anyway. For more information, see https://airbytehq-team.slack.com/archives/C02U9R3AF37/p1730329381148769